src/Algos.cc

```diff
@@ -91,10 +67,9 @@ getCmdOutput(const std::string_view cmd, const usize retry) {
 
 bool
 commandExists(const std::string_view cmd) noexcept {
-  std::string checkCmd = "command -v ";
-  checkCmd += cmd;
-  checkCmd += " >/dev/null 2>&1";
-  return execCmd(checkCmd) == EXIT_SUCCESS;
+  std::string checkCmd = "command -v " + std::string(cmd);
+  int status = std::system(checkCmd.c_str());
```

Comment:
This outputs the result to the terminal, doesn't it? It would be great if we could pipe both stdout and stderr to /dev/null via your Command class.

Reply:
I thought using `which` command is portable enough, but if it isn't, we can:

- use `system()` and `command` command while escaping special characters (by making such function)
- implement `which`-like operation (iterate over PATH and find executable)

What do you think which is preferred?

Reply:
I think `which` is fine for now. Let's revisit this if it becomes an issue.